tests/test_utils.py

```diff
@@ -128,6 +128,11 @@ def test_pack_query_params_one(self):
             falcon.to_query_str({'things': ['a', 'b']}),
             '?things=a,b')
 
+        self.assertEqual(
+            falcon.to_query_str({'things': ['a', 'b', True, False]},
```

Comment:
It may also be helpful to test values that evaluate to `False`, such as `0`, to ensure that in the future someone doesn't inadvertently change the processing logic from the explicit identity checks of `list_value is True` and `list_value is False`, as currently implemented.